File: parantier-front/src/entities/pilot/types/pilotMindmap.ts
export interface PilotMindmap {
  id: number;
  pilotId: number;
  title: string;
  content: string;
  orderNum: number;
  createdAt: string;
  updatedAt: string;
}

export interface CreatePilotMindmapRequest {
  title: string;
  content: string;
  orderNum?: number;
}

export interface UpdatePilotMindmapRequest {
  title: string;
  content: string;
  orderNum: number;
}

File: parantier-api/src/main/java/com/mapo/palantier/pilot/domain/PilotMindmap.java
package com.mapo.palantier.pilot.domain;

import java.time.LocalDateTime;

public class PilotMindmap {
    private Long id;
    private Long pilotId;
    private String title;
    private String content;
    private Integer orderNum;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;

    public PilotMindmap() {}

    public Long getId() { return id; }
    public void setId(Long id) { this.id = id; }

    public Long getPilotId() { return pilotId; }
    public void setPilotId(Long pilotId) { this.pilotId = pilotId; }

    public String getTitle() { return title; }
    public void setTitle(String title) { this.title = title; }

    public String getContent() { return content; }
    public void setContent(String content) { this.content = content; }

    public Integer getOrderNum() { return orderNum; }
    public void setOrderNum(Integer orderNum) { this.orderNum = orderNum; }

    public LocalDateTime getCreatedAt() { return createdAt; }
    public void setCreatedAt(LocalDateTime createdAt) { this.createdAt = createdAt; }

    public LocalDateTime getUpdatedAt() { return updatedAt; }
    public void setUpdatedAt(LocalDateTime updatedAt) { this.updatedAt = updatedAt; }
}
